README.md

```diff
@@ -103,154 +105,56 @@ The process of selecting a best value involves the following steps:
-How will GAM integrate into the final selection layer in the browser and will it help or hinder SSPs? If GAM were to provide an an API/adapter that integrated with Prebid.js this would be a good thing. If, on the other hand, GAM provides a tag library that directly invokes ```navigator.renderInterestGroupAd``` without allowing other SSPs to participate this would be bad.
-
-# An alternative idea that won't work:
-While the TurtleDove author has stated the goal is that TurtleDove execute a local auction over SSP demand ([#73](https://github.com/WICG/turtledove/issues/73)), I don't see how this is possible without provding an API that returns a price signal back to the SSP without rendering an ad. An example of what this might look like:
-
-```javascript
-// This can be thought of as an OpenRTB Bid Response (to the original contextual
-// bid request).
-let sspContextualOrtbBidResp = {
-    "adm": "...creative html...",
-    "price": 1.5,
-    ...
-}
-
-// This could be thought of as an OpenRTR Bid Request, since the meta-data passed 
-// in is likely to be very similar to the original contextual bid request.
-let sspOrtbBidReq = {
-    "site": {
-        "ref": "https://reverb.com/",
-        "publisher": {
-            ...
-        }
-    }
-    ...
-}
-
-// This could be thought of as an OpenRTB Bid Request to an SSP's bid cache on the
-// browser. The browser responds with a structure that includes an opaque pointer
-// (for rendering of the ad later should the winner be an IG ad) and the value
-// of the browser's auction resolution, which would be max(contextual bid, IG bids).
-let sspRealtimeResultPtr = navigator.resolveInterestGroupAd(sspContextualOrtbBidResp, sspOrtbBidReq)
-```
-
-At this point the SSP would know the value they provide to the publisher in realtime and could wrap this logic in their Prebid.js adapter and Prebid.js would function as it does today.
-
-However, TurtleDove views an IG based bid price to be a privacy escape vector so this approach isn't feasible. While this model was a nice thought exercise demonstrating how an SSP might function in a TurtleDove world if an SSP scoped decision was possible, I think we need to assume that it's not.
+17. Ad has rendered
+
+As can be seen in the sequence diagram above, the Parakeet API is invoked by Prebid which is also responsible for constructing the arguments.
+
+## Open Questions
+
+1. Who operates the parakeet server and can all parties trust them?
+The default operator in the proposal seems to be the browser vender, who primarily represents and is trusted by the user. However, Buyers and Sellers will need to trust the Parakeet server to protect their proprietary data from being leaked. The [Garuda](https://darobin.github.io/garuda/) proposal provides a possible governance/trust model but Microsoft has not commited to something like this and it may be up to each browser vendor to make their own decision on how to run the trusted server.
```

Comment:
```suggestion
The default operator in the proposal seems to be the browser vendor, who primarily represents and is trusted by the user. However, Buyers and Sellers will need to trust the Parakeet server to protect their proprietary data from being leaked. The [Garuda](https://darobin.github.io/garuda/) proposal provides a possible governance/trust model but Microsoft has not committed to something like this and it may be up to each browser vendor to make their own decision on how to run a trusted server.
```